packages/desktop_drop/lib/src/channel.dart

```diff
@@ -40,7 +40,6 @@ class DesktopDrop {
   Future<void> _handleMethodChannel(MethodCall call) async {
     switch (call.method) {
       case "entered":
-        assert(_offset == null);
```

Comment:
The assertion here is to guarantee that:
1. The `entered` event will not be received repeatedly.
2. It is no longer in the dropping state, that is, it has received an `exit` or `performOperation` event.